### ISSUE TITLE
Use `seeker.thrust`, not `physical.max_speed`

### ISSUE THREAD

**Issue by Belissimo-T** [seekers-dev/seekers-java#38]

**Comment by Belissimo-T** [comment]
The config should be `seeker.thrust`, not `physical.thrust`.

**Comment by karl-zschiebsch** [comment]
~~~
private double thrust = SeekerProperties.getDefault().getPhysicalThrust();
~~~

This is in physical. Thrust effects Goals and Seekers. Thats the reason why I put it in ``Physical`` instead of ``Seeker``.

**Comment by Belissimo-T** [comment]
@karl-zschiebsch To my knowledge, only seekers can accelerate und thus only seekers are affected by this config.

**Comment by karl-zschiebsch** [comment]
Ok, then I will change that.

**Comment by Belissimo-T** [comment]
@karl-zschiebsch Goals, however, always have a thrust of 0.

**Comment by Belissimo-T** [comment]
fixed in 0e2c177.
